_posts/2020-03-31-ion-hash-js-2_0-released.md

```diff
@@ -0,0 +1,10 @@
+---
+layout: news_item
+title: "Ion Hash JS 2.0 Released"
+date: 2020-03-31
+categories: news

```

Comment:
Does this need an `ion-hash-js` category tag?

Reply:
added, no harm in doing so